lib/wovnrb/url_language_switcher.rb

```diff
@@ -0,0 +1,122 @@
+module Wovnrb
+  # URL Language switching helper, ported from html-swapper
+  class UrlLanguageSwitcher
```

Comment:
Some features was removed during the port such as site-prefix support and custom domain lang support